tigerpath/majors_and_certificates/majors/GER_2018.json

```
@@ -0,0 +1,403 @@
+                  "name": null,
+                  "max_counted": 2,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 3,
+              "min_needed": null,
+              "explanation": "A maximum of three relevant cognate courses in Art & Archaeology, Music, Philosophy, European Cultural Studies, and the Program in Visual Arts.",
+              "course_list": [
+                "ART *",
+                "MUS *",
+                "PHI *",
+                "ECS *",
+                "VIS *"
+              ]
+            }
+          ]
+        },
+        {
+          "name": "German Linguistics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Concentrates on the history and structure of the German language. Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics, and two seminars related to topics in Germanic linguistics: GER 505 (History of the German Language) or GER 508, and GER 506 (Second Language Acquisition and Pedagogy) or GER 316. In addition, such students will take at least three other courses in German literature and culture (usually no more than two at the 200 level), as well as two cognate courses.",
+          "req_list": [
+            {
+              "name": "Linguistics",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics.",
+              "course_list": [
+                "LIN 213",
+              ]
+            },
+            {
+              "name": "Germanic Linguistics",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Two seminars related to topics in Germanic linguistics: GER 505 (History of the German Language) or GER 508, and GER 506 (Second Language Acquisition and Pedagogy) or GER 316.",
+              "req_list": [
+                {
+                  "name": null,
+                  "max_counted": 1,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 505",
+                    "GER 508"
+                  ]
+                },
+                {
+                  "name": null,
+                  "max_counted": 1,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 506",
+                    "GER 316"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "German",

```

```suggestion
              "name": "German literature and culture",
```
https://github.com/TigerPathApp/tigerpath/issues/189#issuecomment-654579852